src/main/java/com/academy/beerinventory/web/controller/BeerController.java

```diff
@@ -31,20 +30,20 @@ public BeerController(IBeerService beerService) {
         return new ResponseEntity<List<BeerDto>>(beersFromService, HttpStatus.OK);
     }
 
-    @PostMapping("/")
+    @GetMapping("/")
```

Comment:
Get mapping de una cerveza debe de pedir el id por la url /{beerId} y 
```suggestion
    @GetMapping("/{beerId}")
```